src/components/NavBar/NavBar.jsx

```diff
@@ -1,33 +1,40 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { ChakraProvider, Button, useDisclosure } from '@chakra-ui/react';
+import UploadCSV from '../UploadCSV/UploadCSV';
 
 import './NavBar.css';
 import FYALogo from '../../assets/fya-logo.png';
 
 function NavBar() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
```

Comment:
Can we rename these variables to be specific to the upload CSV modal? (e.g. isUploadCSVOpen)